src/Components/Navigation.js

```diff
@@ -18,6 +20,7 @@ const Navigation = () => (
         </Link>
       </li>
     </ul>
+    <FaUser className="user-icon" style={{ color: '#0290ff', cursor: 'pointer' }} />
```

Comment:
- [ ] Inline styles are not allowed you can use the React inline styles by adding the styles into a variable and passing it into the component/tag, kindly implement this so your code follows the requirements
e.g:
```
const myComponentStyle = {
  color: 'blue',
  lineHeight: 10,
  padding: '1.5em',
}

function MyComponent(){

    return <div style={myComponentStyle}> Inline Styled Component</div>

}
```